pkg/controller/job/job_controller.go

```diff
@@ -442,15 +451,15 @@ func (jm *Controller) updateJob(old, cur interface{}) {
 			total := time.Duration(*curADS) * time.Second
 			// AddAfter will handle total < passed
 			jm.queue.AddAfter(key, total-passed)
-			klog.V(4).Infof("job %q ActiveDeadlineSeconds updated, will rsync after %d seconds", key, total-passed)
+			klog.FromContext(ctx).V(4).Info("Job's ActiveDeadlineSeconds updated, will rsync", "job", klog.KObj(curJob), "interval", total-passed)
```

Comment:
nit: suggesting using `key` instead of `klog.KObj(curJob)` since already at hand, and to be consistent with other places like this: `klog.FromContext(ctx).Info("Enqueueing job", "job", key)`.